GETTING_STARTED.md

```diff
@@ -1,109 +1,86 @@
 # Getting Started
 
-The SDK supports three ways for authentication:
-
-1. Using the email and password:
+## Auth - Connect to GDN
+The first step to start using GDN is establishing a connection to [gdn.paas.macrometa.io](https://gdn.paas.macrometa.io/). When this code executes, it initializes the server connection to your nearest region. You can access your Macrometa GDN account using several methods, such as [API keys](https://macrometa.com/docs/account-management/api-keys/), [User authentication](https://macrometa.com/docs/account-management/auth/user-auth), [Token-based authentication (JWT)](https://macrometa.com/docs/account-management/auth/jwts). For this example the method used is API key.
 
 ```python
-  # Authentication email and password
-  client = C8Client(protocol='https', host='gdn.macrometa.io', port=443,
-   email="user@example.com", password="XXXXX")
+from c8 import C8Client
+client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443, apikey=<your api key>)
 ```
+## Create a collection
+A [Collection](https://macrometa.com/docs/collections/) consists of documents, and they can be local(only stores data in one region) or global(replicates data and maintains consistency across regions). Macrometa offers several types of collections such as [Document Store](https://macrometa.com/docs/collections/documents/), [Key-Value Store](https://macrometa.com/docs/collections/keyvalue/), [Dynamo Table](https://macrometa.com/docs/collections/dynamo/create-dynamo-table) and [Graph-Edge collection](https://macrometa.com/docs/collections/graphs/). For this example we are going to create a [Document Store](https://macrometa.com/docs/collections/documents/).
 
-2. Using bearer token:
+```python
+client.create_collection(name='employees', sync=False, edge=False, local_collection=False, stream=False)
+```
+# CRUD Operations for collections
+## Insert Document
+Documents are stored in collections. Macrometa GDN is schemaless, so you do not need to define valid document attributes when creating your collection. Documents with completely different structures can be stored in the same collection.
```

Comment:
I think somewhere from CRUD operations for collections till this Insert document line we need to mention that the following operations are being performed on a Document Store or Document collection...??? Since it creates confusion at least in my mind..